src/shared/schemas.ts

```diff
@@ -140,41 +139,42 @@ export class SchemaService {
  * @param extensionContext VSCode extension context
  */
 export async function getDefaultSchemas(extensionContext: vscode.ExtensionContext): Promise<Schemas | undefined> {
-    const cfnSchemaUri = vscode.Uri.joinPath(extensionContext.globalStorageUri, 'cloudformation.schema.json')
-    const samSchemaUri = vscode.Uri.joinPath(extensionContext.globalStorageUri, 'sam.schema.json')
     const devfileSchemaUri = vscode.Uri.joinPath(extensionContext.globalStorageUri, 'devfile.schema.json')
-
-    const goformationSchemaVersion = await getPropertyFromJsonUrl(goformationManifestUrl, 'tag_name')
     const devfileSchemaVersion = await getPropertyFromJsonUrl(devfileManifestUrl, 'tag_name')
 
+    // Sam schema is a superset of Cfn schema, so we can is it for both
+    const samAndCfnSchemaDestinationUri = vscode.Uri.joinPath(extensionContext.globalStorageUri, 'sam.schema.json')
+    const samAndCfnCacheKey = 'samAndCfnSchemaVersion'
+    const samAndCfnSchemaUrl = `https://raw.githubusercontent.com/aws/serverless-application-model/main/samtranslator/schema/schema.json`
```

Comment:
Note for testing purposes only: the schema in `main` doesn't yet include any docs while the one in `develop` (https://github.com/aws/serverless-application-model/blob/develop/samtranslator/schema/schema.json; about 3x larger) does. So documentation should show up upon hover for that one. `main` will be updated over coming weeks.

Reply:
Thanks for pointing this out, I wasn't sure at first if documentation just wasn't included when I was playing around with it. I'll use `develop` for now.